design/pipeline-artifacts.md

```diff
@@ -37,23 +39,80 @@ Artifacts are distinct from other `resources` types, including `containers`, `re
 
 `download` is a shortcut for the [Download Pipeline Artifacts](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/download-pipeline-artifact) task.
 
-It will download artifacts uploaded from a previous job or stage or from another pipeline.
+It will download artifacts uploaded from a previous job, stage, or from another pipeline.
 
 ### Artifact download location
 
-Artifacts are downloaded either to `$PIPELINE_RESOURCESDIRECTORY` or to the directory specified in `root`. Each artifact is given its own directory e.g. `$PIPELINE_RESOURCESDIRECTORY\default` for the `default` artifact of this pipeline. Artifacts coming from other Pipelines are each given one directory per pipeline e.g. `$PIPELINE_RESOURCESDIRECTORY\some-other-pipeline\default` for the `default` artifact of the `some-other-pipeline` pipeline.
+#### YAML / unified pipelines
+Artifacts are downloaded to the pipeline's workspace by default. We introduce a new variable for unified pipelines: `$(Pipeline.Workspace)`. `Pipeline.Workspace` is one level up from `$(System.DefaultWorkingDirectory)`; on hosted, it corresponds to `c:\agent\_work\1\`.
```

Comment:
On hosted this corresponds to D:\a\1

Reply:
Removed, since that was not adding much anyhow.